src/main/java/dataMgmt/FileContentWriter.java

```diff
@@ -1,10 +1,11 @@
 package dataMgmt;
 
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 import patching.Patch;
 import patching.PatchManager;
 
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
 import java.io.*;
```

Comment:
imo java.* stuff should go above any other 3rd party stuff, and then your own should come last. so like 
```Java
import java.*
...

import org.*
...

import patching.*
...
```